extended/src/main/java/apoc/vectordb/Milvus.java

```diff
@@ -133,7 +133,7 @@ public Stream<EmbeddingResult> get(@Name("hostOrKey") String hostOrKey,
                                          @Name("collection") String collection,
                                          @Name("ids") List<Object> ids,
                                          @Name(value = "configuration", defaultValue = "{}") Map<String, Object> configuration) throws Exception {
-        return getCommon(hostOrKey, collection, ids, configuration, true);
```

Comment:
why is this Mode.WRITE? (can you also check the others)?

@Procedure(value = "apoc.vectordb.milvus.get", mode = Mode.WRITE)
    @Description("apoc.vectordb.milvus.get(hostOrKey, collection, ids,